bundles/org.openhab.binding.dbquery/README.md

```diff
@@ -0,0 +1,214 @@
+# DBQuery Binding
+
+This binding allows creating items from the result of native database queries.
+It currently supports InfluxDB 2.0 and will support major relational databases through JDBC.
```

Comment:
If you want to release it without JDBC support first, better don't mention it here.

Reply:
Ok, I will remove it. 
If I found some time, and now that you are reviewing it, and I hope it can be accepted after making desired corrections. I hope to work on the SQL binding soon but I think it's better to be able to integrate current work just In case I found little time and it comes later.

In fact, I want to try TimescaleDB for measurements and will need to implement that part because being able to integrate iotawatt readings is a must for me.